e2e/pages/caseView.page.js

```diff
@@ -172,4 +172,17 @@ module.exports = {
   seeInCaseTitle(titleValue) {
     I.seeElement(locate(this.caseTitle).withText(titleValue));
   },
+
+  formatCaseId(caseId) {
```

Comment:
Maybe double check with Chak/Vijay who are also tidying up flaky tests. Think Vijay has written a function in his recent review which does this too.

Reply:
I think I will reuse Vijay's function after his deployment.